tests/test_connections/test_oef/test_models.py

```diff
@@ -54,6 +54,7 @@ def test_description(self):
         attribute_bar = Attribute("bar", str, True, "a bar attribute.")
         data_model_foobar = DataModel("foobar", [attribute_foo, attribute_bar], "A foobar data model.")
         description_foobar = Description({"foo": 1, "bar": "baz"}, data_model=data_model_foobar)
+        description_foobar.__iter__()
```

Comment:
why is this needed?

Reply:
I manually call this in order to check that returns a description. I probably should test it in some different way that I don't know yet. I tried to change it to `myDescription = iter(description_foobar)` but I get an error.